axum-macros/src/lib.rs

```diff
@@ -37,7 +37,7 @@
     missing_debug_implementations,
     missing_docs
 )]
-#![deny(unreachable_pub, private_in_public)]
+#![deny(unreachable_pub)]
```

Comment:
Did you mean to remove `private_in_public`? I like having that 🤔

Reply:
The [compiler complained](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=45633c5e4070cc44032892418cb25545) about that lint being removed:

```
warning: lint `private_in_public` has been removed: replaced with another group of lints, see RFC <https://rust-lang.github.io/rfcs/2145-type-privacy.html> for more information
 --> src/main.rs:1:9
  |
1 | #![deny(private_in_public)]
  |         ^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(renamed_and_removed_lints)]` on by default
```

I can readd that if you want but it seems not to be helpful having these warnings in all the compile test outputs.

Reply:
Ah I see. Thats probably okay. axum-macros just exposes proc-macros anyway.